<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="smartGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(89, 89, 89);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>71</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Color Type</string>
    </property>
    <widget class="QRadioButton" name="radioButton_rgb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_cmy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CMY</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_hsv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_xyz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>XYZ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_cielab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CIELab</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_yuv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>YUV</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_gray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Pseudo Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionImport_photo_from_Simple_Processing"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionImport_photo_from_Simple_Processing">
   <property name="text">
    <string>Import photo from Simple Processing</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>smartGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>smartgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
